### ISSUE TITLE
🐛 BUG: `<Markdown/>` in SSR mode not working as expected

### ISSUE THREAD

**Issue by thepassle** [withastro/astro#3098]
### What version of `astro` are you using?

1.0.0-beta.8

### Are you using an SSR adapter? If so, which one?

netlify

### What package manager are you using?

npm

### What operating system are you using?

mac

### Describe the Bug

I'm using the `<Markdown/>` component in my SSR project, which locally works fine, but when I deploy to netlify I get the following error message when trying to load the page:

<img width="849" alt="image" src="https://user-images.githubusercontent.com/17054057/163143179-b18496ef-5f53-4a17-b3ac-de2ac20c6eeb.png">

```
{
"errorType": "Error",
"errorMessage": "Cannot find package '@astrojs/markdown-remark' imported from /var/task/netlify/functions/entry.js",
"trace": [
"Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@astrojs/markdown-remark' imported from /var/task/netlify/functions/entry.js",
"    at new NodeError (internal/errors.js:322:7)",
"    at packageResolve (internal/modules/esm/resolve.js:732:9)",
"    at moduleResolve (internal/modules/esm/resolve.js:773:18)",
"    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:887:11)",
"    at Loader.resolve (internal/modules/esm/loader.js:89:40)",
"    at Loader.getModuleJob (internal/modules/esm/loader.js:242:28)",
"    at Loader.import (internal/modules/esm/loader.js:177:28)",
"    at importModuleDynamically (internal/modules/cjs/loader.js:1028:27)",
"    at exports.importModuleDynamicallyCallback (internal/process/esm_loader.js:30:14)",
"    at Object.value (/var/task/netlify/functions/entry.js:91371:13)"
]
}
```

I tried to install `@astrojs/markdown-remark` as a dependency, rebuild and redeploy, but it seemed to not make any difference.

I tried to make a reproduction, but as mentioned locally, and apparently also on stackblitz it seems to work fine. The error only shows up when deployed to netlify.

![image](https://user-images.githubusercontent.com/17054057/163143133-1baad92a-112f-43f4-b179-8a5f2493c881.png)

As an aside: the astro.new stackblitz demo doesnt seem to work anymore:
![image](https://user-images.githubusercontent.com/17054057/163143364-d633c7ff-ffd7-4cfb-8bfd-25642582879b.png)

### Link to Minimal Reproducible Example

no

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by matthewp** [comment]
Have a PR up for this: https://github.com/withastro/astro/pull/3036. but there's a dependency that doesn't bundle well. I'll see if I can get this over the finish line.

**Comment by matthewp** [comment]
This should be working now, let me know if you find out otherwise

**Comment by thepassle** [comment]
Under which package/version is this released? `astro@1.0.0-beta.10`?

**Comment by danielfilho** [comment]
I’m having this exact same issue with gitpod link listed for the blog example on astro.new — here: https://astro.new/blog?on=gitpod
